docs/index.html

```diff
@@ -209,7 +209,14 @@ <h1>Style Guide</h1>
                         <div class="h2 underline-primary">
                             Getting started
                         </div>
-                        <p>...</p>
+                        <p>This is the style guide for all Zeppelin Labs related projects, built on Twitter Bootstrap 4
+                        To use it, navigate with the menu to the left or scroll down below
+
+                        </p>
```

Comment:
Move the closing tag back to the last word of the paragraph (without empty lines).